testsuite/features/init_clients/allcli_update_activationkeys.feature

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) 2022-2023 SUSE LLC
 # Licensed under the terms of the MIT license.
 
+@skip_if_github_validation
```

Comment:
I don't think there is a real need for this tag here, as we never trigger this feature in our run_sets yaml files on the context of github_validation.
But I can understand it as a valid safety net, just in case.

Reply:
Strange then that Jordi has put it on every scenario of the end of that file.